spec/openapi.yaml

```diff
@@ -0,0 +1,271 @@
+openapi: 3.0.1
+info:
+  title: PrusaLink Web
+  description:
+    This is a printer webservice.
+  contact:
+    email: link@prusa3d.cz
+  license:
+    name: AGPLv3
+  version: 1.0.0-draft
+externalDocs:
+  description: Find out more about API
+  url: http://connect.prusa3d.cz
+security:
+  - digestAuth: []
+paths:
+  /api/version:
+    get:
+      summary: api version informations
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Version"
+
+  /api/v1/info:
+    get:
+      summary: printer informations
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Info"
+        401:
+          $ref: "#/components/responses/Unauthorized"
+
+components:
+  schemas:
+    Version:
+      type: object
```

Comment:
I believe I've asked to have some kind of „features“ field here. One that could contain stuff like:

* This printer has a MMU, that MMU has 5 slots and is currently turned off.
* This printer has a modular bed, so there may be multiple temperatures.
* We have 4 extruders, each one with its own temperature.

Also, how do we recognize if this „new“ API is supported or octoprint-ish legacy one should be used?

Reply:
Information which you has better place - /api/v1/info - not version, which is for something different